pkg/patch/patch.go

```diff
@@ -11,16 +11,20 @@ import (
 const (
 	ecrSecret        = `{"name": "appmesh-ecr-secret"}`
 	create           = `{"op":"add","path":"/spec/%v", "value": [%v]}`
+	createElement    = `{"op":"add","path":"/spec/%v", "value": %v}`
 	add              = `{"op":"add","path":"/spec/%v/-", "value": %v}`
 	createAnnotation = `{"op":"add","path":"/metadata/annotations","value":{"%s": "%s"}}`
 	updateAnnotation = `{"op":"%s","path":"/metadata/annotations/%s","value":"%s"}`
+
+	defaultFSGroup int64 = 1337
```

Comment:
It doesn't really have to be an `int64`, but it doesn't matter. Also, we could have a comment describing why we don't want it configurable, but that's not essential either.

Reply:
yeah, it don't need to be int64 since it's used as a string :D, but i used int64 to align with k8s's spec.
Added comments on why we don't want it configurable :D